Xamarin.Forms.Platform.WPF/CellControl.cs

```diff
@@ -85,6 +85,27 @@ void SetSource(object oldCellObj, object newCellObj)
 				Content = null;
 		}
 
+		protected override void OnRenderSizeChanged(SizeChangedInfo sizeInfo)
+		{
+			if (Content is ViewCell vc)
+			{
+				if (vc.LogicalChildren != null && vc.LogicalChildren.Any())
+				{
+					foreach (var child in vc.LogicalChildren)
+					{
+						if (child is Layout<View> layout)
+						{
+							if (layout.HorizontalOptions.Expands)
+							{
```

Comment:
Why only if this ? we should layout always  no ?

Reply:
Probably - I should have seen that.  I can fix and test this evening. 

Thanks for the review

Reply:
i still don't get why we just call layout if `layout.HorizontalOptions.Expands` ?

Reply:
Good question.. my assumptions may not be valid. Should the child layout of a cell always expand to fit the cell?

Reply:
We have to always layout content. I have fixed it in my PR.